product-list.html

```diff
@@ -0,0 +1,284 @@
+                    <label for='price-desc' class='text-capital'>high to low</label>
+                </div>
+            </div>
+        </div>
+        
+        <div class='margin-btm-md flex-column'>
+                <p class='font-size-md text-capital letter-space-md margin-btm-sm'>
+                    brands
+                </p>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input class='margin-right-md' type='checkbox' id='nike-category' />
+                    <label for='nike-category' class='text-capital'>nike</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input class='margin-right-md' type='checkbox' id='adidas-category' />
+                    <label for='adidas-category' class='text-capital'>adidas</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input class='margin-right-md' type='checkbox' id='converse-category' />
+                    <label for='converse-category' class='text-capital'>converse</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input class='margin-right-md' type='checkbox' id='nbalance-category' />
+                    <label for='nbalance-category' class='text-capital'>new balance</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input class='margin-right-md' type='checkbox' id='vans-category' />
+                    <label for='vans-category' class='text-capital'>vans</label>
+                </div>
+        </div>
+        
+        <div class='margin-btm-md flex-column'>
+            <p class='font-size-md text-capital letter-space-md margin-btm-sm'>rating</p>
+            <div id="rating-group">
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' id='4-atleast' name="rating-group" class='margin-right-md' />
+                    <label for='4-atleast' class="text-capital">atleast 4 stars</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' class='margin-right-md' name="rating-group" id='3-atleast' />
+                    <label for='4-atleast' class="text-capital">atleast 3 stars</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' class='margin-right-md' id='2-atleast' name="rating-group"/>
+                    <label for='4-atleast' class="text-capital">atleast 2 stars</label>
+                </div>
+                <div class='flex-vertical-center margin-btm-sm'>
+                    <input type='radio' class='margin-right-md' name="rating-group" id='1-atleast' />
+                    <label for='4-atleast' class="text-capital">atleast 1 star</label>
```

Comment:
Change the for attribute in labels to their corresponding inputs. Also try to keep the attributes in same order in all the inputs.

Reply:
thanks for pointing out @anik31 . i have updated the code.